content/docs/05.developer-guide/02.tasks.md

```diff
@@ -486,6 +518,27 @@ tasks:
         console.log(colors.red("Hello"));
 ```
 
+This task can also fetch files from [namespace files][namespace files](./namespace-files.md) and make them available for all sub-tasks.
+
+```yaml
+id: node-with-cache
+namespace: io.kestra.tests
+
+tasks:
+  - id: working-dir
+    type: io.kestra.core.tasks.flows.WorkingDirectory
+    namespaceFiles:
+      include: 
```

Comment:
Do we need to add `enabled: true` or not? if not, then ✅ 
But it seems it's by default set to false so the filter wouldn't work without `enabled: true`